### PR TITLE
Compile all language support in one python file

### LINKED ISSUE #3: Compile all language support in one python file.
The current structure of project provides different files for a different format of the logic file, merge them into one single file with if blocks.

### PR DESCRIPTION
**Pull request by agarwalrounak**
### Checklist:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you successfully ran tests with your changes locally?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### Issues Solved

Fixes #3 

### Changes:
- Compiled all language support in one python file(for both Python2 and Python3)
- Made the code PEP8 compatible
- Updated README accordingly

### CODE REVIEW

File: Python2/Example/TCGen.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    os.mkdir('input')
+    os.mkdir('output')
+except OSError:
+    pass
+
+pwr = math.pow
+rint = random.randint
+
+choice = raw_input("Enter your choice of language\n1 for C\n2 for C++\n3 for Java\n4 for Python")
+if choice != '1' or choice != '2' or choice != '3' or choice != '4':
```
Comment (aashutoshrathi) [inline Python2/Example/TCGen.py]: This logic is wrong, makes it:
```py
if choice != '1' and choice != '2' and choice != '3' and choice != '4':
```

File: Python2/Example/TCGen.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+try:
+    os.mkdir('input')
+    os.mkdir('output')
+except OSError:
+    pass
+
+pwr = math.pow
+rint = random.randint
+
+choice = raw_input("Enter your choice of language\n1 for C\n2 for C++\n3 for Java\n4 for Python")
```
Comment (aashutoshrathi) [inline Python2/Example/TCGen.py]: Add "\n" at the end once more.

File: Python2/TC-Generator/TCGen.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import shutil
+
+try:
+    os.mkdir('input')
+    os.mkdir('output')
+except OSError:
+    pass
+
+choice = raw_input("Enter your choice of language\n1 for C\n2 for C++\n3 for Java\n4 for Python")
+if choice != '1' or choice != '2' or choice != '3' or choice != '4':
```
Comment (aashutoshrathi) [inline Python2/TC-Generator/TCGen.py]: Same changes here too.

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Check comments and test it locally.

**Comment by agarwalrounak** [comment]
Done. Please review.

**Review by aashutoshrathi — COMMENTED** [review]
Yeah, that thing works fine now.
Make choice field an integer only.

**Comment by agarwalrounak** [comment]
@aashutoshrathi done, please review.

**Review by aashutoshrathi — APPROVED** [review]
LGTM Great work.

**Comment by agarwalrounak** [comment]
Thanks! Please merge the PR.

**Pull request merged**